kolibri/plugins/learn/assets/src/vue/index.vue

```diff
@@ -4,7 +4,10 @@
 
     <nav id="learn-nav">
       <ul>
-        <a v-link="{ name: 'learn-page' }">
+        <a
+          v-link="{ name: $options.PageNames.LEARN_ROOT }"
```

Comment:
I am having trouble understanding how what I presume is the `constants` module is getting passed into the options here?

Wouldn't it be better to directly require the `constants` here rather than rely on them being passed down in the options?